draft-ietf-mls-architecture.md

```diff
@@ -696,6 +696,109 @@ software is updated while the client is a member of a group. Thus, in addition
 to preventing downgrade attacks, the members of a group can also observe when it
 is safe to upgrade to a new ciphersuite or protocol version.
 
+# Interoperability Requirements
+
+A fully functional deployment of MLS requires a number of infrastructure
+services to be available, as well as several policies and design trade-offs that
+are decided by the application. This section lists all of the dependencies of an
+MLS deployment that are external to the protocol specification, but would still
+need to be aligned for two deployments to potentially interoperate.
+
+MLS has the built-in ability to negotiate protocol versions, ciphersuites,
+extensions, and credential types. For two deployments to interoperate, they must
```

Comment:
`Capabilities` also has proposal types.

Reply:
It might be worth mentioning that `required_capabilities` can help ensure that things fail cleanly when required things aren't supported.